app/renderer/reducers/frameReducer.js

```diff
@@ -117,22 +117,32 @@ const frameReducer = (state, action, immutableAction) => {
         state = state.setIn(['frames', index, 'title'], title)
       }
 
-      // TODO fix race condition in Muon more info in #9000
-      const active = immutableAction.getIn(['tabValue', 'active'])
+      // Unlike tabValue, active state for changeInfo will be fired only once
+      // when a tab is set to active, and will not be updated for the same tab
+      // as long as it remains active.
+      // This likely deprecates tabValue but still lacks state for fresh profile.
+      // TODO check if changeInfo is a better fit than tabValue and make use
+      // of it app-wise and deprecate the latter.
+      const active = immutableAction.getIn(['changeInfo', 'active'])
+
+      // This is a workaround for changeInfo which do not provide
+      // active state on a fresh profile.
+      if (active == null) {
```

Comment:
Actually we want to check to make sure that `changeInfo` is not null. If changeInfo.active is null that just means that the active status hasn't changed